.devops/deploy-pipelines.yml

```diff
@@ -107,18 +128,14 @@ stages:
               tags: |
                 $(Build.BuildId)
```

Comment:
Same as before for the name of the branch

Reply:
the branch name in this case will be always "main"